docs/data/material/experimental-api/css-variables/css-variables.md

```diff
@@ -196,6 +198,62 @@ export function onRenderBody({ setPreBodyComponents }) {
 }
 ```
 
+### TypeScript
+
+You need to import the theme augmentation to include `theme.vars` and other utilities related to CSS variables to the theme:
+
+```ts
+// this can be the root file of you application
+import type {} from '@mui/material/themeCssVarsAugmentation';
+```
+
+Then, you will be able to access `theme.vars` in any of the styling APIs, for example the `styled`:
+
+```ts
+import { styled } from '@mui/material/styles';
+
+const StyledComponent = styled('button')(({ theme }) => ({
+  // typed-safe
+  color: theme.vars.palette.primary.main,
+}));
+```
+
+#### Augment palette types
+
+The module augmentation approach remains the same. For example, if you want to add new tokens to the palette you can do this:
```

Comment:
Let's link to the other page that goes into details for how to override the palette values.

Reply:
I think may be better to remove this section.